tpr/tpr.go

```diff
@@ -0,0 +1,120 @@
+package tpr
+
+import (
+	"fmt"
+	"time"
+
+	microerror "github.com/giantswarm/microkit/error"
+	"github.com/rossf7/pingdom-operator/pkg/util"
```

Comment:
https://github.com/giantswarm/operatorkit/pull/9/commits/d5fc356dccd0581fd6406eedf04a7422e552602d

I feel that operator should fail after a while if it can't create TPR. I don't think it's possible with backoff without hacks. LMK if you think different.